### PR TITLE
Add helm new function support yaml 10390

### PR DESCRIPTION
**Pull request by nitind123**
Add helm new function support yaml 10390

### CODE REVIEW

**Review by neha-mishraa — CHANGES_REQUESTED** [review]
can you squash all commits

File: helmFile10390-1/.jfrog-pipelines/.jfrog-pipelines/P_SYC_Pipelines_10390_file.yml [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+  {{ end -}}
 pipelines:
   - name: P_files
     steps:
       - name: S_files
         type: Bash
         configuration:
           environmentVariables:
-            env1: gitBranch
+            env1: 'gitbranch'
```
Comment (Varsha-Arun) [inline helmFile10390-1/.jfrog-pipelines/.jfrog-pipelines/P_SYC_Pipelines_10390_file.yml:11]: why did we add value in quotes

File: tests/uploadTemplate/templates/jfrog/helmTemplate/values.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+GitRepo:
+  name: fooGitRepo
+  gitProvider: s_gitHub
+  path: jfbetapipeorg/Org_test_Automation
+  branch: master
+  branches:
+    exclude: 'test'
```
Comment (Varsha-Arun) [inline tests/uploadTemplate/templates/jfrog/helmTemplate/values.yml:7]: why exclude in quotes and not include

**Review by Varsha-Arun — APPROVED** [review]

**Comment by nitind123** [comment]
@neha-mishraa - done ...please merge

**Review by neha-mishraa — APPROVED** [review]

**Pull request merged**